### PR TITLE
to do list structure

### PR DESCRIPTION
**Pull request by adarachel**
Project requirements
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader

### CODE REVIEW

**Review by Whoistolu — COMMENTED** [review]
Hi @adarachel 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file. Kindly ensure that you use the professional [README template](https://github.com/microverseinc/readme-template) recommended by Microverse and fill in every section of it before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,185 @@
...
+
+I would like to thank and appreciate who contributes this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is under [MIT](./LICENSE) license.
```
Comment (omarsalem7) [inline README.md:183]: - [ ] Kindly fix the broken MIT license link and add your own license

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  color: purple;
+  font-size: larger;
+}
+
+.clear button {
+  color: white;
+  font-size: large;
+  font-weight: 800;
+  border: none;
+  width: 100%;
```
Comment (omarsalem7) [inline src/style.css]: - [ ] Kindly consider styling your project according to the style of the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). And consider using the same colors on it

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @adarachel 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Use webpack to bundle JavaScript. ✔️
- Use ES6 modules to write modular JavaScript.✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (adarachel) [reply src/style.css]: i do not know the exact colors used in the video

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @adarachel 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ correct use of GitHub flow
✔️ linters are passing
✔️ professional documentation
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**